snekbox/nsjail.py

```diff
@@ -129,16 +137,24 @@ def _consume_stdout(self, nsjail: subprocess.Popen) -> str:
         return "".join(output)
 
     def python3(
-        self, code: str, *, nsjail_args: Iterable[str] = (), py_args: Iterable[str] = ("-c",)
-    ) -> CompletedProcess:
+        self,
+        py_args: Iterable[str],
+        files: Iterable[FileAttachment] = (),
+        *,
```

Comment:
The placement of this `*` seems arbitrary. It would make sense for it to be before `py_args`, but I would be fine if it was just removed entirely at this point.

Reply:
Removed `*` in [4787a94](https://github.com/python-discord/snekbox/pull/159/commits/4787a9497be4f460e028031ddad4cd9d8388b84f)